examples/rds/main.tf

```diff
@@ -37,7 +39,7 @@ module "example_datastore_rds" {
   source = "../../"
 
   providers {
-    aws = "aws"
+    aws = "aws.ap-southeast-2"
```

Comment:
doesn't this change make it more work for someone to run the example in any other region?

Reply:
Yes, Which gives me an idea to try out. :)

Reply:
I have made some changes that I feel make it cleaner to set a region for running the examples. Will discuss this before committing it.

Reply:
Implemented vars.tf using `my_region` patterns to simplify example usage. Great recognition of ease of use @rdunn-Hypr